.github/CONTRIBUTING.md

```diff
@@ -12,53 +11,44 @@ Support KB staff members review issues and pull requests on a regular basis. We
 Support KB works with Magento Community Engineering teams and projects.
 As you contribute PRs, you gain [Contribution Points](../docs/contribution-points.md).
 
-
 ## Get started
 
-
 ![Get started workflow](../docs/img/contributor_start.png)
 
-1. Make sure you have a [GitHub account](https://github.com/signup/free).
-
-    **Note for partners:** Add [2FA](https://devdocs.magento.com/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
-
-1. [Fork](https://help.github.com/articles/fork-a-repo/) the [Support KB repository](https://github.com/magento/knowledge-base). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
-1. Review the [Support KB guidelines below](#contribution-guidelines).
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork](https://help.github.com/articles/fork-a-repo/) the [Support KB repository](https://github.com/magento/knowledge-base). Remember to [sync your fork](https://help.github.com/articles/syncing-a-fork/) and update branches as needed.
+* Review the [Support KB guidelines below](#contribution-guidelines).
 
+>**Note for partners:** Add [2FA](https://devdocs.magento.com/contributor-guide/contributing.html#two-factor) protection when contributing to Magento repositories.
 
 ## Contribute
 
 The following diagram shows the contribution workflow:
 
 ![Contributing workflow](../docs/img/Contributor-PR.png)
 
-
 ### Create a branch
 
-1. Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
-1. Edit or create markdown (`.md`) files in your branch.
-1. Push your branch to your fork.
+* Create a new branch from your fork using a name that best describes the work or references a GitHub issue number.
+* Edit or create markdown (`.md`) files in your branch.
+* Push your branch to your fork.
 
 ### Create a pull request
 
-1. Create a pull request to the [magento/knowledge-base](https://github.com/magento/knowledge-base) repository. Use `main` as the base branch when creating a PR.
-
-1. Complete the pull request providing the information listed in the template.
+* Create a pull request to the [magento/knowledge-base](https://github.com/magento/knowledge-base) repository. Use `main` as the base branch when creating a PR.
+* Complete the pull request providing the information listed in the template.
+  - **We will close your pull request if you do not provide the information described in the template.**
```

Comment:
Mixing unordered styles - you can't do this. It's '*' or '-'
Plus second level list items need to be indented by 3 spaces/

Reply:
Thank you So much, Barny Shergold, I tried What many times still these linting issues are not getting solved. Please review it once and suggest a possible solution 🙌